utils/RebalancingTokenWrapper.ts

```diff
@@ -47,27 +49,47 @@ export class RebalancingTokenWrapper {
     factory: Address,
     transferProxy: Address,
     tokenCount: number,
+    naturalUnits: BigNumber[] = undefined,
     from: Address = this._tokenOwnerAddress,
   ): Promise<SetTokenContract[]> {
-    const naturalUnit = ether(2);
+    let naturalUnit: BigNumber;
     const setTokenArray: SetTokenContract[] = [];
 
     const components = await this._erc20Wrapper.deployTokensAsync(tokenCount + 1, this._tokenOwnerAddress);
     await this._erc20Wrapper.approveTransfersAsync(components, transferProxy);
 
     const indexArray = _.times(tokenCount, Number);
     for (const index in indexArray) {
+      let minDec: number;
```

Comment:
Let's favor being verbose and descriptive in variable names. 

`minimumDecimal` vs `minDec`